frontend/views/components/AvatarGenerator.vue

```diff
@@ -0,0 +1,48 @@
+<template lang='pug'>
+  // Leave in the DOM for debug
```

Comment:
Do you want to keep this now?

Reply:
At least during CR so @mmbotelho can see it too easily :)